### PR TITLE
validation for Ov6.3

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
validation for Ov6.3

### Issues Resolved
NA

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG file.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,53 @@ Extends support of the SDK to OneView REST API version 3200 (OneView v6.30)
 - [#173] (https://github.com/HewlettPackard/oneview-python/issues/73) is this a typo https://github.com/HewlettPackard/oneview-python/blob/master/examples/server_hardware.py#L200-201
 
 ##### Features supported with the current release
```
Comment (chebroluharika) [inline CHANGELOG.md:8]: In first line change unreleased to released

Comment (AsisBagga) [reply CHANGELOG.md:8]: we are not releasing this yet. Its just the validation.

Comment (chebroluharika) [reply CHANGELOG.md:8]: okay

**Review by chebroluharika — APPROVED** [review]

File: endpoints-support.md [hunk 2]
```diff
@@ -18,259 +18,259 @@
...
-|<sub>/rest/appliance/network-interfaces/{macAddress}</sub>                                 |   GET        |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  | :white_check_mark:   |
-|<sub>/rest/appliance/network-interfaces/mac-addresses</sub>                                |    GET       |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  | :white_check_mark:   |
+|<sub>/rest/appliance/network-interfaces</sub>                                              |    GET       |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
```
Comment (yuvirani) [inline endpoints-support.md:31]: Added twice. Please correct

Comment (AsisBagga) [reply endpoints-support.md]: done

**Review by yuvirani — APPROVED** [review]

**Pull request merged**